src/harness/sas.py

```diff
@@ -135,6 +135,15 @@ def Relinquishment(self, request, ssl_cert=None, ssl_key=None):
   def Deregistration(self, request, ssl_cert=None, ssl_key=None):
     return self._CbsdRequest('deregistration', request, ssl_cert, ssl_key)
 
+  def ImplementationRecordExchange(self, request, ssl_cert=None, ssl_key=None):
+    return self._SasRequest('sas', request, ssl_cert, ssl_key)
```

Comment:
'sas' has changed to 'sas_impl' in the latest version of the spec (and our API doc).